source/common/quic/platform/quiche_stack_trace_impl.h

```diff
@@ -8,11 +8,44 @@
 
 #include <sstream>
 #include <string>
+#include <vector>
 
 #include "source/server/backtrace.h"
 
+#include "absl/debugging/stacktrace.h"
+#include "absl/debugging/symbolize.h"
+
 namespace quiche {
 
+// NOLINTNEXTLINE(readability-identifier-naming)
+inline std::vector<void*> CurrentStackTraceImpl() {
```

Comment:
This seems reasonable, but I'm curious where it came from. Did you write this from scratch? I wonder if we could use the default platform impl of CurrentStackTraceImpl?

Reply:
I took this implementation from the upstream QUICHE change, although https://github.com/envoyproxy/envoy/blob/main/source/server/backtrace.h uses some of the same underlying utilities and constant values.